subprojects/language-native/src/integTest/groovy/org/gradle/language/AbstractNativeLanguageIncrementalBuildIntegrationTest.groovy

```diff
@@ -320,6 +321,7 @@ abstract class AbstractNativeLanguageIncrementalBuildIntegrationTest extends Abs
     }
 
     @Requires(TestPrecondition.CAN_INSTALL_EXECUTABLE)
+    @RequiresInstalledToolChain(SUPPORTS_32_AND_64)
```

Comment:
❓ What is the default requirements? Does it assumes host architecture?

Reply:
The assumption is that a toolchain supports a single architecture, which is usually the default architecture of the host (64-bit), but for windows gcc we currently do some magic to make it see the default architecture as 32-bit.  The new requirements should be used wherever a test requires both 32 bit and 64 bit, or wherever a test explicitly declares a 32-bit architecture.